openwisp_controller/config/tests/test_admin.py

```diff
@@ -914,6 +914,37 @@ def test_clone_template(self):
         response = self.client.post(path, data, follow=True)
         self.assertContains(response, '{} (Clone 3)'.format(t.name))
 
+    def test_get_template_default_values(self):
+        t1 = self._create_template(name='t1', default_values={'name1': 'test1'})
+        path = reverse('admin:get_template_default_values')
+
+        with self.subTest('get default values for one template'):
+            r = self.client.get(path, {'pks': f'{t1.pk}'})
```

Comment:
can you add an assertion using `assertNumQueries` here? To keep the amount of queries in check in the future in case someone changes the code.